code/datums/spells/cluwne.dm

```diff
@@ -0,0 +1,88 @@
+/obj/effect/proc_holder/spell/targeted/touch/cluwne
+	name = "Curse of the Cluwne"
+	desc = "Turns the target into a fat and cursed monstrosity of a clown."
+	hand_path = /obj/item/weapon/melee/touch_attack/cluwne
+
+	school = "transmutation"
+
+	charge_max = 600
+	clothes_req = 1
+	cooldown_min = 200 //100 deciseconds reduction per rank
+
+	action_icon_state = "clown"
+
+/mob/living/carbon/human/proc/makeCluwne()
+	to_chat(src, "<span class='danger'>You feel funny.</span>")
+	adjustBrainLoss(80)
+	nutrition = 9000
+	overeatduration = 9000
+	confused = 120
```

Comment:
This may be a bit much for our codebase.

On TG/Goon confusion doesn't guarantee that the tile you step on will be messed up, on ours, it does, so this is pretty much 2 minutes of walking around in a circle. IMO, make this 30; that's still a full minute of walking around in a circle, dizzy and is still very debilitating.